easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-5.1.1-2023a.yml

```
@@ -10,3 +10,7 @@ easyconfigs:
         # See https://github.com/easybuilders/easybuild-easyconfigs/pull/22932
         from-commit: 3211d34eb16ff31b8de3dfef55ecaaf1ec205c6f
   - MrBayes-3.2.7-gompi-2023a.eb
+  - Flye-2.9.3-GCC-12.3.0.eb
+      options:
+        # See https://github.com/easybuilders/easybuild-easyconfigs/pull/20004
+        from-commit: cccb85fc90b3df1e00be5bf8131f857ff5d92faa
```

This version is included with EB 5.1.1, so there's no need for `from-commit`.
```suggestion
  - Flye-2.9.3-GCC-12.3.0.eb
```